### DISCUSSION TITLE
Vscode showing subscription required after I have subscribed

### DISCUSSION THREAD

**Discussion by Charpell** [community/community#30432] [Copilot Conversations]
I subscribed for the monthly plan, after restarting my vscode i still see this badge showing I can't use the service
![Screenshot 2022-08-22 at 18 27 10](https://user-images.githubusercontent.com/21138774/185982704-ffc90496-8216-48ce-b60a-386129d9b594.png)

**Comment by bortvern** [comment]
I am experiencing this same behavior. I have two Github accounts, one personal and one for a business I'm working with. I have a subscription under my personal account, but so far am unable to associate Visual Studio code with the correct subscription.

**Comment by jarmo-seljamaa** [comment]
I subscribed for the yearly plan and have one single GitHub account, but still the same problem: VS Code shows "Subscription Required"

**Comment by oscarBack** [comment]
> I subscribed for the monthly plan, after restarting my vscode i still see this badge showing I can't use the service ![Screenshot 2022-08-22 at 18 27 10](https://user-images.githubusercontent.com/21138774/185982704-ffc90496-8216-48ce-b60a-386129d9b594.png)

same problem ... two github accounts and the one subscribed require "activation" ... github said 30 minutes to "take effect", but already been 1 hour
![image](https://user-images.githubusercontent.com/74551171/186018519-f3150ad2-4084-4c85-9875-46e13dba9719.png)
